Stopping: 'install' failed for 'Alien::libmaxminddb'.

C:\>cpan install Alien::libmaxminddb
Loading internal logger. Log::Log4perl recommended for better logging
CPAN: CPAN::SQLite loaded ok (v0.219)
Database was generated on Wed, 14 Sep 2022 01:08:16 GMT
Running install for module 'Alien::libmaxminddb'
CPAN: Digest::SHA loaded ok (v6.02)
CPAN: Compress::Zlib loaded ok (v2.096)
Checksum for C:\strawberry\cpan\sources\authors\id\V\VO\VOEGELAS\Alien-libmaxminddb-1.007.tar.gz ok
CPAN: Archive::Tar loaded ok (v2.38)
CPAN: YAML::XS loaded ok (v0.82)
CPAN: CPAN::Meta::Requirements loaded ok (v2.140)
CPAN: Parse::CPAN::Meta loaded ok (v2.150010)
CPAN: CPAN::Meta loaded ok (v2.150010)
CPAN: Module::CoreList loaded ok (v5.20200717)
Configuring V/VO/VOEGELAS/Alien-libmaxminddb-1.007.tar.gz with Makefile.PL
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::PP
Alien::Build> error in probe (will do a share install): package libmaxminddb not found at C:/Strawberry/perl/vendor/lib/Alien/Build/Plugin/PkgConfig/PP.pm line 85.
Alien::Build> Don't panic, we will attempt a share build from source if possible.
Alien::Build> Do not file a bug unless you expected a system install to succeed.
Checking if your kit is complete...
Looks good
Generating a gmake-style Makefile
Writing Makefile for Alien::libmaxminddb
Writing MYMETA.yml and MYMETA.json
  VOEGELAS/Alien-libmaxminddb-1.007.tar.gz
  C:\Strawberry\perl\bin\perl.exe Makefile.PL -- OK
Running make for V/VO/VOEGELAS/Alien-libmaxminddb-1.007.tar.gz
cp lib/Alien/libmaxminddb.pm blib\lib\Alien\libmaxminddb.pm
"C:\Strawberry\perl\bin\perl.exe" -MAlien::Build::MM=cmd -e prefix site C:\strawberry\perl\lib C:\strawberry\perl\site\lib C:\strawberry\perl\vendor\lib
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::PP
main> prefix C:/strawberry/perl/site/lib/auto/share/dist/Alien-libmaxminddb
"C:\Strawberry\perl\bin\perl.exe" -MAlien::Build::MM=cmd -e version 1.007
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::PP
"C:\Strawberry\perl\bin\perl.exe" -MAlien::Build::MM=cmd -e download
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::PP
Alien::Build::Plugin::Core::Download> decoding html
no matching files in listing at C:/Strawberry/perl/vendor/lib/Alien/Build/Plugin/Core/Download.pm line 30.
gmake: *** [Makefile:1018: _alien/mm/download] Error 9
  VOEGELAS/Alien-libmaxminddb-1.007.tar.gz
  C:\strawberry\c\bin\gmake.exe -- NOT OK
Stopping: 'install' failed for 'Alien::libmaxminddb'.

--- a/alienfile
+++ b/alienfile
@@ -4,11 +4,8 @@ use Path::Tiny qw(path);
 plugin 'PkgConfig' => 'libmaxminddb';
 
 share {
-  start_url 'https://github.com/maxmind/libmaxminddb/releases';
-  plugin 'Download' => (
-    filter => qr/^libmaxminddb-[0-9\.]+\.tar\.gz$/,
-    version => qr/^libmaxminddb-([0-9\.]+)\.tar\.gz$/,
-  );
+  start_url 'https://github.com/maxmind/libmaxminddb/releases/download/1.6.0/libmaxminddb-1.6.0.tar.gz';
+  plugin 'Download';
   plugin 'Extract' => 'tar.gz';
   plugin 'Build::Autoconf';
   patch \&patch_pkgconfig;

